### PR TITLE
fix: [connect] Move devDependencies to dependencies

### PR DESCRIPTION
**Pull request by clayne11**
The package is unable to function at runtime without these dependencies, so they're actually runtime dependencies, not devDepedencies.

### CODE REVIEW

File: connect/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "default": "./dist/*.js"
     }
   },
-  "devDependencies": {
+  "dependencies": {
```
Comment (StarpTech) [inline connect/package.json:22]: `typescript` and `de-cli` are dev dependencies for sure.

**Review by StarpTech — CHANGES_REQUESTED** [review]
Good finding. I made a comment because not all of them are prod deps.

File: connect/package.json (hunk 1 above)
Comment (clayne11) [reply connect/package.json:22]: Fixed.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**